books_options.rb

```diff
@@ -0,0 +1,26 @@
+require './book'
+
+class BooksOption
+  attr_reader :books
+
+  def initialize
+    @books = []
+  end
+
+  # List all books
+  def list_all_books
+    @books.each_with_index do |book, index|
+      puts "#{index + 1})  Title: #{book.title}, Author: #{book.author}"
+    end
+  end
```

Comment:
- [ ] (Optional) It would be nice to print a message that says no book is available if a `book` has not been created. 👍